pkg/operator/ceph/csi/spec.go

```diff
@@ -359,7 +363,8 @@ func (r *ReconcileCSI) startDrivers(ver *version.Info, ownerInfo *k8sutil.OwnerI
 			return errors.Wrap(err, "failed to load rbdplugin template")
 		}
 		if tp.CSILogRotation {
-			tp.CSILogFolder = "rbd-plugin"
+			tp.CsiComponentName = nodePlugin
+			tp.CsiLogRootPath = csiRootPath + "/" + RBDDriverName
```

Comment:
Instead of a string append with `/`, use `path.Join()` wherever combining paths
```suggestion
			tp.CsiLogRootPath = path.Join(csiRootPath, RBDDriverName)
```

Reply:
Done, thanks
Wasnt knowing such function exists